frontend/src/components/Thread.vue

```diff
@@ -53,16 +69,34 @@ export default {
     },
   },
   methods: {
+    // TODO: this is currently not used, because I can't figure out a good place to put this
     resolve(thread) {
       this.$store.commit('resolveThread', thread);
     },
     reply(thread) {
-      console.log(thread);
-      this.$store.commit('addMessageToThread', [this.newMessages[thread.id], thread]);
-      this.newMessages[thread.id] = "";
+      if (this.newMessages[thread.id]) {
+        console.log(thread);
```

Comment:
do we need this console.log? if we do can you add a prefix to specify what file it's coming from

Reply:
I don't think we need this one.